NOTICE

```diff
@@ -0,0 +1,2 @@
+rsmt2d
+Copyright 2018 and onwards Mustafa Al-Bassam
```

Comment:
[question] out of curiosity why is this one Mustafa Al-Bassam instead of Strange Loop Labs AG?